### PR TITLE
feat(base): creates project base and server

### PR DESCRIPTION
**Pull request by sjl2**
- Created Hapi server
- Started Server without errors
- Installed and started nodemon for server changes

### CODE REVIEW

**Comment by charlesbjohnson** [comment]
You should add a `.gitignore` file to exclude some unwanted files from version control. The `node_modules` folder, specifically, is one that we tend not to check into git but there are others as well. Take a look at how [lob-api's `.gitignore`](https://github.com/lob/lob-api/blob/master/.gitignore) is setup.

After you've done that, you'll need to remove the unwanted files from your commit. You can do this by undoing your commit locally, unstaging all files, staging the files you actually want to commit, and then making a new commit. Finally, you'll need to force push to GitHub with your updated branch (since you'll have re-written history, so to speak). I recommend taking a look at [git-tips](https://github.com/git-tips/tips) to see how you can do some of that git magic.

Feel free to ping me if you have questions.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    "eslint": "^1.9.0",
+    "eslint-config-lob": "^2.0.0",
+    "generate-changelog": "^1.0.0",
+    "istanbul": "^0.4.2",
+    "mocha": "^2.2.5",
+    "nodemon": "^1.9.2"
+  },
+  "dependencies": {
+    "hapi": "^13.5.0"
+  }
```
Comment (elnaz) [inline package.json:37]: nit: It's the generators fault (https://github.com/lob/generator/issues/11) but could you move the `dependencies` above `devDependencies` and order the npm `scripts` alphabetically to keep the `package.json` consistent with our other repositories?

File: lib/server.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  connections: {
+    router: { stripTrailingSlash: true } // removes trailing slashes on incoming paths
+  }
+}); 
+
+server.connection({ port: Config.PORT });
+
+module.exports = server; 
+
+
```
Comment (elnaz) [inline lib/server.js]: nit: extra new line

File: lib/server.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const Hapi = require('hapi');
+
+const Config = require('../config');
+
+const server = new Hapi.Server({
+  connections: {
+    router: { stripTrailingSlash: true } // removes trailing slashes on incoming paths
```
Comment (elnaz) [inline lib/server.js]: You can remove my comment from miyagi here

**Comment by elnaz** [comment]
LGTM. Now that you've seen the failing build, you can remove the `npm test` and `npm run enforce` lines from your `.travis.yml` until you add tests in a later pr.

File: lib/index.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const Util = require('util');
+
+const Config = require('../config');
+const Movies = require('./server');
+
+Movies.start(() => {
+  Util.log(`A different server started on port: ${Config.PORT}`);
```
Comment (charlesbjohnson) [inline lib/index.js]: A different server ❓

Comment (charlesbjohnson) [inline lib/index.js]: ❄️

**Comment by charlesbjohnson** [comment]
LGTM!

**Pull request merged**